cvat/apps/engine/media_extractors.py

```diff
@@ -662,7 +662,12 @@ def save_as_chunk(self, images, chunk_path):
                 output = io.BytesIO()
                 if self._dimension == DimensionType.DIM_2D:
                     pil_image = rotate_within_exif(Image.open(image))
-                    pil_image.save(output, format=pil_image.format if pil_image.format else self.IMAGE_EXT, quality=100, subsampling=0)
+                    # https://pillow.readthedocs.io/en/stable/handbook/image-file-formats.html
+                    if pil_image.format == 'TIFF':
+                        # use loseless lzw compression for tiff images
+                        pil_image.save(output, format='TIFF', compression='tiff_lzw')
+                    else:
+                        pil_image.save(output, format=pil_image.format if pil_image.format else self.IMAGE_EXT, quality=95, subsampling=0)
```

Comment:
Why quality 95?

Reply:
Values above 95 should be avoided; 100 disables portions of the JPEG compression algorithm, and results in large fileswith hardly any gain in image quality.

https://jdhao.github.io/2019/07/20/pil_jpeg_image_quality/

In my test file size with quality 100 was twice more than with quality 95.

Reply:
It is not the original file in the case. Can we try to save in PNG format in the case and use the original name?

Reply:
There are some libraries which allow to rotate a jpeg file with 90 degrees step without loosing quality.

Reply:
Okay, I will return 100 quality. 

> There are some libraries which allow to rotate a jpeg file with 90 degrees step without loosing quality.

This is not only about jpeg images, exif is about different formats.